freestyle-effects/shared/src/test/scala/effects/EffectsTests.scala

```diff
@@ -223,6 +222,102 @@ class EffectsTests extends AsyncWordSpec with Matchers {
     }
   }
 
+  "Validation integration" should {
+    import freestyle.effects._
+    import freestyle.effects.validation._
+    import freestyle.effects.validation.implicits._
+
+    import cats.data.{State, StateT}
+    import cats.instances.future._
+
+
+    type Logger[A] = StateT[Future, Errors, A]
+
+    case object MissingFirstName extends ValidationException {
+      val explanation = "The first name is missing"
+    }
+
+    "valid" in {
+      def program[F[_]: ValidationM] =
+        for {
+          _ <- Applicative[FreeS[F, ?]].pure(1)
+          b <- ValidationM[F].valid(42)
+          _ <- Applicative[FreeS[F, ?]].pure(1)
+        } yield b
+
+      program[ValidationM.Op].exec[Logger].run(List.empty) map {  _ shouldBe Tuple2(List(), 42) }
```

Comment:
FWIW, we could replace these `.run(List.empty)` with `.runEmpty` if we `import cats.instances.list._`.